internal/proto/data_coord.proto

```diff
@@ -45,6 +48,7 @@ service DataCoord {
   rpc SetSegmentState(SetSegmentStateRequest) returns (SetSegmentStateResponse) {}
   // https://wiki.lfaidata.foundation/display/MIL/MEP+24+--+Support+bulk+load
   rpc Import(ImportTaskRequest) returns (ImportTaskResponse) {}
+  rpc UpdateSegmentStatistics(UpdateSegmentStatisticsRequest) returns (Empty) {}
```

Comment:
Looks strange if an grpc will never returns error. Say `datacoord` is not healthy or segment does not exist, an error is needed to notify either the system or the user.

Reply:
Done.